added titles to the docs pages (#109)
dappforce/subsocial-docs: docs/develop/sdk/reactions/FetchReactions.md

title: Fetch Reactions
displayed_sidebar: developSidebar
---

<head>

>   <title>Fetch Reactions on the Subsocial Blockchain | JS SDK Guide</title>

Reviewer: ```suggestion
  <title>How To Get Reactions With The Subsocial JS SDK</title>
```
